samples/bluetooth/broadcast_audio_sink/Kconfig

```diff
@@ -35,4 +35,12 @@ config TARGET_BROADCAST_NAME
 	  Name of target broadcast device. If not empty string, sink device
 	  will only listen to the specified broadcast source. Not case sensitive.
 
+config ENABLE_LC3
+	bool "Enable the LC3 codec"
+	# By default let's enable it in the platforms we know are capable of supporting it
+	default y
+	depends on (ARCH_POSIX || SOC_SERIES_NRF53X)
```

Comment:
Does this also enable it for the nRF5340 netcore?

Reply:
"No, but" (only if you were to build this application itself into the netcore, which would not be what someone would do normally, or what they would do if they used sysbuild)

Reply:
Just wondering if it would make more sense to depend on `SOC_NRF5340_CPUAPP` instead

Reply:
done